packages/components/src/utils/console.ts

```diff
@@ -0,0 +1,19 @@
+const red = "\u001b[31m "
+const yellow = "\u001b[33m "
+const reset = "\u001b[0m "
+
+export const error = (message: string): Error => {
+  throw new Error(
+    `${red}\nCULTUREAMP UI ERROR:\n${singleLine(message)}${reset}\n`
+  )
+}
+
+export const warn = (message: string): void => {
```

Comment:
Looks like we only use `warn` (so the others aren't needed), and I'm sure we had previous feedback that these console warnings were just noisy and unappreciated.
Maybe just make this a `Modal` util to keep it contained.